ui/app/controllers/allocations/allocation/task/fs.js

```diff
@@ -20,6 +20,16 @@ export default Controller.extend({
   directories: filterBy('directoryEntries', 'IsDir'),
   files: filterBy('directoryEntries', 'IsDir', false),
 
+  pathWithLeadingSlash: computed('path', function() {
+    const path = this.path;
+
+    if (path.startsWith('/')) {
+      return path;
+    } else {
+      return `/${path}`;
```

Comment:
We might also want to check that it doesn't start with `[A-Z]:\` too? Not sure how the FS api works on windows tho tbh

Reply:
hmm good question, thanks. Is there some cloud-y deployment of Nomad I can use? Or maybe I can set it up locally in a virtual machine 😯

Reply:
A local vm is probably your best bet right now :disappointed:

Reply:
I finally managed it. It looks like `\`s are abstracted away:

![image](https://user-images.githubusercontent.com/43280/62155537-c3d47a80-b2ce-11e9-9cab-31f5058169da.png)

I was able to query `server\empty.txt` and get the same response. So maybe I don’t have to care about `\`s 🤔

Reply:
:tada: